modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsRelationshipsDisplayContext.java

```diff
@@ -116,6 +124,68 @@ public long getObjectDefinitionId() {
 		return objectDefinition.getObjectDefinitionId();
 	}
 
+	public ArrayList<JSONObject> getObjectRelationshipDeletionTypes() {
+		ArrayList<JSONObject> deletionTypes = new ArrayList<>();
+
+		deletionTypes.add(
+			JSONUtil.put(
```

Comment:
I see that you populate your list with this sequence: prevent > disassociate > cascade. Does this order matter? If not, we should sort this list.